src/main/app/controller/courts/AddressController.ts

```diff
@@ -0,0 +1,276 @@
+import autobind from 'autobind-decorator';
```

Comment:
Do you think it would be worth creating a new class(es) for the address and postcode validation under a utils folder or something? It's all really good stuff! Maybe it can be reused if we need it again, or perhaps it would reduce the size of the controller class a bit :- )

Reply:
Good call. I've moved the postcode validation to the existing validation utils class. I've left the address validation in the controller because I'm not sure it'll have any reuse.